### DISCUSSION TITLE
Require MUI style switch in Paste

### DISCUSSION THREAD

**Discussion by ShaunHalliday** [twilio-labs/paste#2199] [Ideas] [answered]
Hi All,

We are currently developing systems for clients using Paste (and other Twilio services) and to keep to the Paste standard we require a switch style component. I cannot currently find anything in Paste to match this requirement.

Looking at Flex 2.0 preview this still uses a MUI switch as shown in below image but require a Paste component: -

<img width="330" alt="Flex2 0 switch using MUI" src="https://user-images.githubusercontent.com/85227673/153198672-19d24b59-8b26-4c48-b4cf-93b136b748ea.png">

One use case is shown below, currently shown as checkboxes but the requirement is for a switch element/component: -

![switch-required-1](https://user-images.githubusercontent.com/85227673/153198885-65287a63-69f3-49c0-ba21-b9ac244c8759.png)

Another use case is the following image, rather than having to click a checkbox, and then save this change, the client requires a switch element/component:-

<img width="136" alt="Switch-required-2" src="https://user-images.githubusercontent.com/85227673/153199391-ef804491-329f-4a82-a08d-8c4076aea448.png">

Please let me know if you require any further information :)

**Comment by serifluous** [comment]
Hey @ShaunHalliday, thanks so much for reaching out! Our team's still taking a look at your request.

In the meantime, are you willing to share more about which other Twilio services you're developing with? Is it Paste and Flex only, or other products, too?

**Reply by ShaunHalliday** [reply]
Hey @serifluous, I work for a gold partner company in the UK so we use the whole suite of Twilio products :)

**Comment by gloriliale** [comment, marked as answer]
Hi @ShaunHalliday! We're undergoing roadmap reviews at the moment, which determine how we prioritize work on upcoming components like the toggle/switch component. We'll have more information about this when the process concludes in a few weeks time. In the meantime, if a toggle/switch is something that's needed urgently, it's possible to make it from the Box primitive.

We'll update this when there's more information. Many thanks!
